docs/content/concepts/layer/control/catalog/_index.md

```diff
@@ -85,157 +88,271 @@ See [examples](#content-examples) of OSCAL catalogs.
+
+{{<highlight xml "nowrap=false" >}}
+<prop name="label" value="AC-1"/>
+<prop name="label" class="sp800-53a" value="AC-01"/>
+<prop name="sort-id" value="ac-01"/>
+{{< / highlight >}}
+
+In this representation of AC-1, three properties are given by the data publisher. Each of these properties may be fairly considered to be a "convenience value": for example, two properties provides data consumers with reliable values for a control label "as published" in different renditions of the catalog. An extra identifier is also provided for sorting. As shown here, `<prop>` elements can also be qualified by `class` to offer further distinctions from neighbor properties.
+
+In OSCAL, the values of properties must be atomic values, such as strings, numbers or dates, without Markdown (for example) or other kinds of formatting. To extend properties to provide for values with more complex object types represents a challenge to be met, because any representation of data at this level must be in a form as widely accessible as possible across the greatest range of consumers (languages, platforms, serialization formats, and information modeling and programming technologies and paradigms). To mitigate this limitation, where simple qualification by assigning name/value pairs is not enough, controls (and other structures) in OSCAL can also be extended using other generic elements such as `<link>` and `<part>`.
+
+#### Links
+
+Following any property sequence come any links proper to this control. Consistently with modern practice, only outbound links are encoded explicitly.
+
+Like parameters and properties, `<link>` elements provide for explicit representation of data given implicitly in published sources. Here, this information is represented by the __Related Controls__ and __References__ sections of the control text. In each case, targets of the links form distinct classes of their own -- they are either cross-referenced controls or enhancements (also `<control>` elements in OSCAL), or references to bibliographic entries listed as back matter. These are distinguished by `rel` values permitting each to be resolved and handled correctly.
+
+On links, `href` follows the semantics of `href` as defined for HTML/HTTP by W3C. `rel` is constrained as an enumerated value by OSCAL.
```

Comment:
Minor, non-blocking: maybe we add a link to W3C, [perhaps this](https://html.spec.whatwg.org/multipage/links.html#introduction-2)?